### PR TITLE
feat(dashboard): render dashboard page and repo cards

### PR DESCRIPTION
**Pull request by KellyMerrick**
Dashboard view with multiple repo cards:

theme-dark
image

theme-light
image

Dashboard with no repos:
image

No dashboards:
image

### CODE REVIEW

**Comment by KellyMerrick** [comment]
Need to add comments still.

**Comment by plyr4** [comment]
overall, epic. excited to see where this goes. 
couple tiny comments

File: src/elm/Components/DashboardRepoCard.elm [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+            , p []
+                [ a
+                    [ class "card-org"
+                    , Route.Path.href <|
+                        Route.Path.Org_
+                            { org = props.card.org
+                            }
+                    ]
+                    [ small [] [ text props.card.org ] ]
+                , br [] []
```
[the remark this answers is not in the document]
Comment (KellyMerrick) [reply src/elm/Components/DashboardRepoCard.elm:135]: keeping as is for now. we have several implementations of similar. perhaps a larger rework?

File: src/elm/Components/RecentBuilds.elm [hunk 2]
```diff
@@ -5,6 +5,8 @@ SPDX-License-Identifier: Apache-2.0
 
 module Components.RecentBuilds exposing (view)
 
+-- import Utils.Errors exposing (showAlertAlways)
```
[the remark this answers is not in the document]
Comment (KellyMerrick) [reply src/elm/Components/RecentBuilds.elm]: removed

File: src/elm/Pages/Dashboards/Dashboard_.elm [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+                                    Http.BadStatus statusCode ->
+                                        case statusCode of
+                                            401 ->
+                                                "Unauthorized to retrieve dashboard"
+
+                                            404 ->
+                                                "Dashboard \"" ++ route.params.dashboard ++ "\" not found. Please check the URL."
+
+                                            _ ->
+                                                "No dashboard found; there was an error with the server"
```
[the remark this answers is not in the document]
Comment (KellyMerrick) [reply src/elm/Pages/Dashboards/Dashboard_.elm]: changed to comma. agree that ; is technically more correct. larger future cleanup?

File: src/elm/Pages/Dashboards/Dashboard_.elm [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+
+                                    _ ->
+                                        "No dashboard found; there was an error with the server"
```
[the remark this answers is not in the document]
Comment (KellyMerrick) [reply src/elm/Pages/Dashboards/Dashboard_.elm]: changed to comma. agree that ; is technically more correct. larger future cleanup?

File: src/scss/_dashboards.scss [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+  gap: 1rem;
+  align-items: center;
+  margin: -0.5rem -0.5rem 0 -0.5rem; // to ignore .card padding
+  padding: 0 1rem;
+
+  line-height: 1.25;
+
+  background-color: var(--color-bg-darkest);
+
+  // custon styling for the status icon
```
[the remark this answers is not in the document]
Comment (KellyMerrick) [reply src/scss/_dashboards.scss]: fixed

File: src/scss/_dashboards.scss [hunk 6]
```diff
@@ -0,0 +1,225 @@
...
+
+  background-color: var(--color-bg-dark);
+}
+
+// a dashboard card header containing org, repo, and status icon
+.card-header {
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+  margin: -0.5rem -0.5rem 0 -0.5rem; // to ignore .card padding
```
[the remark this answers is not in the document]
Comment (KellyMerrick) [reply src/scss/_dashboards.scss:30]: research and @wass3rw3rk discussion; this seems to be best way to break out of parent element styling.

**Review by wass3rw3rk — APPROVED** [review]

**Review by plyr4 — APPROVED** [review]
sweet! thanks for the updates too

**Pull request merged**